L/libCppInterOp/build_tarballs.jl

```diff
@@ -0,0 +1,94 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
+
+name = "libCppInterOp"
+version = v"0.1.0"
+
+llvm_versions = [v"17.0.6"]
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Gnimuc/CppInterOp.git", "416a123977a9bebf0254f2b68a6f48497bc1debc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd CppInterOp/
+mkdir build && cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+     -DLLVM_DIR=${prefix}/lib/cmake/llvm \
+     -DClang_DIR=${prefix}/lib/cmake/clang \
+     -DCMAKE_BUILD_TYPE=Release
+make -j${nproc}
+make install
+install_license ../LICENSE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+filter!(p -> libc(p) != "musl", platforms) # LLVM_full+asserts isn't available for musl
```

Comment:
That's true only for i686-linux-musl, not all musl-based platforms.

Reply:
Fixed!